### ISSUE TITLE
Incompatibility with 8.8.2

### ISSUE THREAD

**Issue by t-hart** [math-comp/math-comp#237]
Hey,

Just wanted to report a seeming incompatibility with Coq 8.8.2 as discovered when trying to update the version on Homebrew from 8.8.1 ([as reported in this pull request](https://github.com/Homebrew/homebrew-core/pull/33451)).  

Is this a known issue? I tried to look for it and #206 seems similar, but I'm not sure it's the same issue.

**Comment by CohenCyril** [comment]
Dear @t-hart, I do not understand what kind of incompatibility you are talking about. Can you detail the process to obtain the failure (without using `brew` if possible)?

**Comment by t-hart** [comment]
Ah, my apologies. It seems the issue was with Homebrew and their process as opposed to with Coq and `math-comp`. 

I've been meaning to get into Coq for a while and checked the Homebrew repo. Realizing it was not up to date, I decided to try and bump the Homebrew version. One of the contributors found the breaking test and said it should be reported upstream with `math-comp`, so I did, but it turns out that it works when rebuilding `math-comp` against 8.8.2.

As such I'll go ahead and close the issue. Sorry for taking up your time.
